lte/protos/session_manager.proto

```diff
@@ -1086,12 +1115,17 @@ service SetInterfaceForUserPlane {
     rpc SendPagingReuest(UPFPagingInfo) returns (magma.orc8r.Void) {}
 }
 
+//SET API for 3 procedures request from AMF to SMF
+service AmfPduSessionSmContext {
+  // PDU session related configuration from amf to smf
+  rpc SetAmfSessionContext(SetSMSessionContext) returns (SmContextVoid) {}
+  // PDU session related notification from amf to smf
+  rpc SetSmfNotification(SetSmNotificationContext) returns (SmContextVoid);    //AMF to SMF
```

Comment:
this comment ` //AMF to SMF` seems redundant

Reply:
resolved